lib/p2p/Pool.ts

```diff
@@ -786,8 +790,12 @@ class Pool extends EventEmitter {
       }
       case PacketType.OrderInvalidation: {
         const orderInvalidation = (packet as packets.OrderInvalidationPacket).body!;
-        this.logger.trace(`received order invalidation from ${peer.label}: ${JSON.stringify(orderInvalidation)}`);
-        this.emit('packet.orderInvalidation', orderInvalidation, peer.nodePubKey as string);
+        if (!this.nodeState.pairs.includes(orderInvalidation.pairId)) {
```

Comment:
Again here it's better to use `peer.isPairActive(pairId)`. Also minor nit but I think the code is a bit easier to read if the if statement checks for the true/active case first, then else prints the inactive pair message.

Reply:
But isn't in this case we'll check incoming peer's active pairs instead of our active pairs?

Reply:
Yes that is true, however I think that's what we want here. If the peer's pair isn't active, then we won't have any orders for it in the order book anyway and the invalidation is moot.